php/add-subscriber-to-list.php

```diff
@@ -0,0 +1,93 @@
+<?php
+  /**
+   * Add a 'member' to a 'list' via mailchimp API v3.x
+   * @ http://developer.mailchimp.com/documentation/mailchimp/reference/lists/members/#create-post_lists_list_id_members
+   *
+   * ================
+   * BACKGROUND
+   * Typical use case is that this code would get run by an .ajax() jQuery call or possibly a form action
+   * The live data you need will get transferred via the global $_POST variable
+   * That data must be put into an array with keys that match the mailchimp endpoints, check the above link for those
+   * You also need to include your API key and list ID for this to work.
+   * You'll just have to go get those and type them in here, see README.md
+   * ================
+   */
+
+  /**
+   * ================
+   * Sets API Key and list ID to add a subscriber
+   * ================
+   */
+  define("api_key", "your-api-key-here");
+  define("list_id", "your-list-id-here");
+
+  /**
+   * ================
+   * DESTINATION URL
+   * Note: your API URL has a location subdomain at the front of the URL string
+   * It can vary depending on where you are in the world
+   * To determine yours, check the last 3 digits of your API key
+   * ================
+   */
+  $url = 'https://us5.api.mailchimp.com/3.0/lists/' . list_id . '/members/';
```

Comment:
I would consider setting the `us5` portion to a variable like `list_id` and `api_key` because it can change from user to user. For example, my list is `us6`